frontend/src/pages/modelRegistry/screens/ModelPropertiesDescriptionListGroup.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import { Button } from '@patternfly/react-core';
+import { Table, Tbody, Th, Thead, Tr } from '@patternfly/react-table';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import text from '@patternfly/react-styles/css/utilities/Text/text';
+import DashboardDescriptionListGroup from '~/components/DashboardDescriptionListGroup';
+import { ModelRegistryCustomProperties } from '~/concepts/modelRegistry/types';
+import ModelPropertiesTableRow from './ModelPropertiesTableRow';
+import { getProperties, mergeUpdatedProperty } from './utils';
+
+type ModelPropertiesDescriptionListGroupProps = {
+  customProperties: ModelRegistryCustomProperties;
+  saveEditedCustomProperties: (properties: ModelRegistryCustomProperties) => Promise<unknown>;
+};
+
+const ModelPropertiesDescriptionListGroup: React.FC<ModelPropertiesDescriptionListGroupProps> = ({
+  customProperties = {},
+  saveEditedCustomProperties,
+}) => {
+  const [editingPropertyKeys, setEditingPropertyKeys] = React.useState<string[]>([]);
```

Comment:
I'm not seeing the "Show more" functionality in Properties section. It's breaking the layout.

<img width="997" alt="Screenshot 2024-05-03 at 6 32 58 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/22885912/e3e17a41-973a-4b13-bc30-bba554e097e0">

Reply:
Resolved, see screenshots in comment below